media/css/plugincheck/plugincheck.less

```diff
@@ -263,8 +263,100 @@ html[dir="rtl"] {
 #sec-plugin-uptodate {
     display: none;
 }
-.footer-newsletter-form {
-    padding: 40px 20px;
+#mozorg-newsletter-form,
+#newsletter-form {
+    width: 1300px / 2;
```

Comment:
This width is making it overflow in narrower windows. I think you don't need a width here at all and could just let it default to auto.
![plugincheck-overflow](https://cloud.githubusercontent.com/assets/205591/10763761/75bc51e0-7c88-11e5-9de5-0c65a01648f6.jpg)